tests/testQuestionParams.js

```diff
@@ -0,0 +1,336 @@
+
+const questionsArray = [addNumbers1_assessment, addNumbers2, addNumbers3, addNumbers4];
+
+describe('Parameterized questions', function () {
+  this.timeout(60000);
+  var page;
+
+  before('set up testing server', helperServer.before());
+  after('shut down testing server', helperServer.after);
+
+  var elemList, res;
+
+  describe('the database', function () {
+    it('should contain questions', function (callback) {
+      sqldb.query(sql.select_questions, [], function (err, result) {
+        if (ERR(err, callback)) return;
+        if (result.rowCount === 0) {
+          return callback(new Error('no questions in DB'));
+        }
+        locals.questions = result.rows;
+        callback(null);
+      });
+    });
+
+    it('should contain the addNumbersParameterized1 question', function () {
+      const question = _.find(locals.questions, { directory: addNumbers1.qid });
+      assert.isDefined(question);
+      addNumbers1.id = question.id;
+      addNumbers1.url = locals.courseBaseUrl + '/question/' + addNumbers1.id + '/';
+      addNumbers1_assessment.url = locals.questionBaseUrl + '/' + addNumbers1.id + '/';
+    });
+    it('should contain the addNumbersParameterized2 question', function () {
+      const question = _.find(locals.questions, { directory: addNumbers2.qid });
+      assert.isDefined(question);
+      addNumbers2.id = question.id;
+      addNumbers2.url = locals.questionBaseUrl + '/' + addNumbers2.id + '/';
+    });
+    it('should contain the addNumbersParameterized3 question', function () {
+      const question = _.find(locals.questions, { directory: addNumbers3.qid });
+      assert.isDefined(question);
+      addNumbers3.id = question.id;
+      addNumbers3.url = locals.questionBaseUrl + '/' + addNumbers3.id + '/';
+    });
+    it('should contain the addNumbersParameterized4 question', function () {
+      const question = _.find(locals.questions, { directory: addNumbers4.qid });
+      assert.isDefined(question);
+      addNumbers4.id = question.id;
+      addNumbers4.url = locals.courseBaseUrl + '/question/' + addNumbers4.id + '/';
+      addNumbers4.url = locals.questionBaseUrl + '/' + addNumbers4.id + '/';
+    });
+  });
+
+  var startAssessment = function () {
+    describe('the locals object', function () {
+      it('should be cleared', function () {
+        for (var prop in locals) {
+          delete locals[prop];
+        }
+      });
+      it('should be initialized', function () {
+        locals.siteUrl = 'http://localhost:' + config.serverPort;
+        locals.baseUrl = locals.siteUrl + '/pl';
+        locals.courseInstanceBaseUrl = locals.baseUrl + '/course_instance/1';
+        locals.questionBaseUrl = locals.courseInstanceBaseUrl + '/instance_question';
+        locals.assessmentsUrl = locals.courseInstanceBaseUrl + '/assessments';
+        locals.isStudentPage = true;
+        locals.totalPoints = 0;
+      });
+    });
```

Comment:
Nit: I suspect you copied something like this from existing tests, but there are no assertions here, so these really shouldn't be in `describe`/`it` blocks.